ng2-components/ng2-alfresco-upload/src/components/file-uploading-dialog.component.ts

```diff
@@ -31,35 +32,51 @@ export class FileUploadingDialogComponent implements OnInit, OnDestroy {
     filesUploadingList: FileModel[] = [];
     isDialogActive: boolean = false;
     totalCompleted: number = 0;
+    totalErrors: number = 0;
     isDialogMinimized: boolean = false;
-    uploadFilesCompleted: boolean = false;
 
     private listSubscription: Subscription;
     private counterSubscription: Subscription;
     private fileUploadSubscription: Subscription;
+    private errorSubscription: Subscription;
 
     constructor(
+        translateService: AlfrescoTranslationService,
```

Comment:
This approach has been deprecated. All i18n resources are initialised on per-module basis now. Please remove "translateService.addTranslationFolder" from this file.